ci/pipeline.yml

```diff
@@ -355,27 +185,27 @@ jobs:
         - put: slack
           params:
             text: |
-              :white_check_mark: SUCCESS: Successfully deployed pages build containers on production
+              :white_check_mark: SUCCESS: Successfully deployed pages build containers on ((deploy-env))
               <$ATC_EXTERNAL_URL/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|View build details>
             channel: ((slack-channel))
             username: ((slack-username))
             icon_url: ((slack-icon-url))
 
-  - name: reconfigure-staging
+  - name: reconfigure-((deploy-env))
```

Comment:
I started calling the job `set-pipeline-((deploy-env))` but just realized we've used the `reconfigure` naming convention before. Whichever we want, let's stick to one.

Reply:
`set-pipeline` seems much clearer - I'll update to that 👍